.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,25 @@
+on: workflow_dispatch
+
+name: Integration tests
+
+jobs:
+  test:
+    name: Integration tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@master
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Decrypt secrets
+        run: integration-tests/decrypt-secrets.sh
+        env:
+          IT_SECRET_PASSPHRASE: ${{ secrets.IT_SECRET_PASSPHRASE }}
+
+      - name: Test
```

Comment:
There is a missing infrastructure spin up part. It should make a terraform apply or destroy and apply on a given workspace.

Reply:
Yep, that will be a part of the #1211 .

Reply:
I'd say that infrastructure should be the first, then manager to spin up scenarios and aggregating results, and then scenarios starting from simpler ones to more complex.

Reply:
One does not depend on the other. We just need the URL of the loadbalancer and grafana to run the tests and gather the metrics.

Reply:
Load balancer and grafana depends on the test cluster. Scenarios cannot be run if there is no infrastructure to run on. Tests cannot be executed if there is no cluster to run on and manager to run from.

Reply:
They're loosely coupled only via the URL. I left the `TODO` to replace the hardcoded hosts IPs with the loadbalancer URL, and I will do it with the next changes.

Reply:
No. Grafana cannot be started without load balancer and load balancer cannot be started without the initial nodes IPs.
Grafana can workaround load balancer but if we would like to use load balancer, terraform should spin up a test cluster, then the load balancer and then the grafana

Reply:
>No.
That's a 'no' to which part, that they're loosely coupled? We just need node IPs, that's all. It doesn't matter if we take it from the load balancer, DNS, or even a static JSON file from S3. There's no need to run terraform apply before the test (unless we want a throw-away environment, that as far as I remember you were against having).

Reply:
They're loosely coupled only via the URL. You don't need to restart the whole infrastructure but you have to spin up an infrastructure

Reply:
Resolved, infrastructure was added in the same workflow.